testsuite/tests/typing-recovery/type_recovery_test_3.compilers.reference

```diff
@@ -0,0 +1,12 @@
+File "_none_", line 1:
+Error: File "type_recovery_test_3.ml", line 12, characters 17-21:
+       12 | let foo2 : int * toto = 3, 4
+                             ^^^^
+       Error: Unbound type constructor toto
+
+
+File "type_recovery_test_3.ml", line 12, characters 17-21:
+12 | let foo2 : int * toto = 3, 4
+                      ^^^^
+Error: Unbound type constructor toto
+
```

Comment:
The error is duplicated / reported twice here.
Why is that?

Reply:
I try to figure why :/ it is very strange.